src/call.js

```diff
@@ -1,14 +1,146 @@
+import {
+  addToPresenceList,
+  hidePresence,
+  setupCreateButton,
+  showCall,
+  showEntry,
+  showPresence,
+} from './dom.js';
+
 window.addEventListener('DOMContentLoaded', () => {
-  const container = document.getElementById('container');
+  setupCreateButton(createRoom);
+
+  // If room URL and name params were provided, join that room
+  const params = new URLSearchParams(window.location.search);
+  const roomURL = params.get('roomURL');
+  const roomName = params.get('roomName');
+  if (roomURL && roomName) {
+    joinRoom(roomURL, roomName);
+    return;
+  }
+
+  // If room URL and name params were not provided, show
+  // entry element (with call creation button)
+  showEntry();
+});
+
+/**
+ * Creates and joins a Daily room
+ * @returns {Promise<void>}
+ */
+async function createRoom() {
+  const url = `/.netlify/functions/createRoom`;
+  const errMsg = 'failed to create room';
 
-  callFrame = window.DailyIframe.createFrame(container, {
+  // Create Daily room
+  let res;
+  try {
+    res = await fetch(url);
+  } catch (e) {
+    console.error('failed to make Daily room creation request', e);
+  }
+
+  // Retrieve body, if it exists in the response
+  let body;
+  try {
+    body = await res.json();
+  } catch (e) {
+    console.warn('No body present in room creation response');
+  }
+
+  // Check status code
+  if (res.status !== 200) {
+    const msg = `${errMsg}. Status code: ${res.status}`;
+    console.error(`failed to create Daily room: ${msg}`, body);
+    return;
+  }
+  joinRoom(body.url, body.name);
+}
+
+/**
+ * Joins a Daily room and retrieves its existing participants
+ * @param roomURL
+ * @param roomName
+ */
+function joinRoom(roomURL, roomName) {
+  showCall();
+
+  // Set up the call frame and join the call
+  const callContainer = document.getElementById('callFrame');
+  const callFrame = window.DailyIframe.createFrame(callContainer, {
     showLeaveButton: true,
-    iframeStyle: {
-      position: 'fixed',
-      width: 'calc(100% - 1rem)',
-      height: 'calc(100% - 5rem)',
-    },
   });
-  // TODO: Replace the following URL with your own room URL.
-  callFrame.join({ url: 'https://<your-domain>.daily.co/<room-name>' });
-});
+  callFrame.on('joined-meeting', () => {
+    // Presence view is no longer needed once the participant is in the room.
+    hidePresence();
+  });
+  callFrame.join({ url: roomURL });
+
+  updateInviteURL(roomURL, roomName);
+
+  // Fetch existing participants and show them next to the call frame
+  fetchParticipants(roomName);
```

Comment:
You'll need to also call this for the `participant-joined` event. 
Steps to recreate the bug I just hit:
- I clicked Create Room (person1)
- in the lobby, saw "Nobody's here".
- clicked the invite URL to join from another tab (person2)
- looked at the original tab (person1) after having person2 join and it still says nobody's here.

This text should update when people join the call after I've joined the lobby (or prejoin, whatever we're calling it)

Reply:
The design was actually to have the user refresh the page to see an updated participant list (so equivalent behavior to Google Meet, for example). Since `"participant-joined"` is not emitted while the user is in the Prebuilt lobby. There are ways to make this more of a live-refresh situation, but I think for this initial implementation I'll keep things basic and just tell them to refresh - then do the fancier version in a followup PR with its own post. 

I'll add a "Refresh to update" notice here to make the behavior clearer.